docs/game-mechanics/behaviors/basic-attack.rst

```diff
@@ -7,33 +7,48 @@ Parameters
 ----------
 
 .. list-table ::
-   :widths: 15
+   :widths: 15 30
    :header-rows: 1
 
    * - Name
+     - Description
    * - dir_angle_xz
+     - The direction to modify the attackers angle by?  Currently unused in Darkflame Universe.
    * - dir_angle_y
+     - The direction to modify the attackers angle by?  Currently unused in Darkflame Universe.
    * - dir_force
+     - The amount of force to apply towards angle specified in :samp:`dir_angle_xz` or :samp:`dir_angle_y`?  Currently unused in Darkflame Universe.
    * - dont_apply_immune
+     - Whether or not to apply immunity to the caster of the behavior.  Currently unused in Darkflame Universe.
    * - max damage
+     - The maximum amount of damage to be dealt.
    * - min damage
+     - The minimum amount of damage to be dealt.
    * - on_fail_armor
+     - The behavior to use on failure due to armor.  Currently unused in Darkflame Universe.
    * - on_fail_blocked
+     - The behavior to use on failure due to the attack being blocked.  Currently unused in Darkflame Universe.
    * - on_fail_immune
+     - The behavior to use on failure due to the target being immune.  Currently unused in Darkflame Universe.
    * - on_success
+     - The behavior to use on success of the attack.
    * - radius
+     - The radius of the attack.  This is likely deprecated as only `one behavior <https://explorer.lu/skills/69>`_ currently uses this.  Currently unused in Darkflame Universe.
    * - use_caster_velocity
+     - Whether or not to use the casters velocity.  Currently unused in Darkflame Universe.
    * - velocity_multiplier
+     - Value to multiply velocity by when attacking.  Currently unused in Darkflame Universe.
 
-BitStream Serialization
+BitStream Serialization for Darkflame Universe
```

Comment:
don't think that's what I meant... This isn't something DLU specific. just the always writing 001 for blocked, immune and such is.

Reply:
Addressed this in the new commit.  Added further notes on the behavior as well :)